.github/workflows/build_image_ci.yml

```diff
@@ -0,0 +1,17 @@
+name: build-image-ci
+
+on: pull_request
+
+env:
+  NODE_VERSION: 18.17.0
+  PHP_VERSION: 8.2
+
+jobs:
+  build-image-ci-job:
```

Comment:
rename the job as 'build-image'  
and try setting the strategy matrix to test the build on 8.1 and 8.2